### DISCUSSION TITLE
Turn off TNT Protection?

### DISCUSSION THREAD

**Discussion by eightballz** [GriefPrevention/GriefPrevention#1629] [Q&A]
I use a plugin called UltimateTNT, But TNT won't explode even when I do /claimexplosins. I **NEED** Explosins on for this gamemode! Is there any possible way to make it so that I can disable it as a whole in the config?

**Comment by TehAwesomestKitteh** [comment]
Set `BlockSurfaceCreeperExplosions` and `BlockSurfaceOtherExplosions` to `false` in `config.yml`. If the region where TNT is being exploded in is claimed land, then also set `BlockLandClaimExplosions` to `false`
